components/notification-bar/notification-bar--scripted.html

```diff
@@ -0,0 +1,41 @@
+<p>{% render '@button', { class: 'mzp-js-notification-trigger', label: 'Open Notification' } %}</p>
+
+<script src="{{ '/protocol/js/protocol-notification-bar.js' | path }}"></script>
+
+<script>
+  (function () {
+    'use strict';
+
+    var dissmissButtons = document.querySelectorAll('.mzp-c-notification-bar-button');
```

Comment:
Typo, remove extra "s"
```suggestion
    var dismissButtons = document.querySelectorAll('.mzp-c-notification-bar-button');
```